### PR TITLE
[Merged by Bors] - chore: replace 'import Batteries' with finer imports

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

**Comment by mattrobball** [comment]
bors delegate+

**Comment by semorrison** [comment]
bors merge

**Comment by semorrison** [comment]
bors merge

**Comment by semorrison** [comment]
bors merge

File: .github/workflows/bors.yml [hunk 1]
```diff
@@ -248,7 +248,10 @@ jobs:
         uses: liskin/gh-problem-matcher-wrap@v3
         with:
           linters: gcc
-          run: lake test
+          run: |
+            # make sure everything is available for import_all.lean
```
Comment (grunweg) [inline .github/workflows/bors.yml:252]: There's no file `import_all.lean` in mathlib (and I don't find the relation to `mk_all` obvious; that doesn't require building batteries). Would you like to clarify this comment?
(The same applies to the three other copies of this comment.)

Comment (semorrison) [reply .github/workflows/bors.yml:252]: `tests/import_all.lean`. I'll update the comment.

**Comment by semorrison** [comment]
bors merge

File: .github/workflows/bors.yml (hunk 1 above)
Comment (grunweg) [reply .github/workflows/bors.yml]: :bulb: Thanks!

**Pull request closed without merging**